client/src/app/privacy/privacy.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  moduleId: module.id,
+  selector: 'op-privacy',
+  templateUrl: 'privacy.component.html',
+  styleUrls: ['privacy.component.css']
```

Comment:
@eyalcohen4 The `css` file is empty. So, if we don't use it, delete it and delete from here.